src/win/fs.c

```diff
@@ -1941,6 +1941,110 @@ INLINE static void fs__stat_prepare_path(WCHAR* pathw) {
   }
 }
 
+INLINE static int fs__stat_directory(WCHAR* path, uv_stat_t* statbuf,
+    int do_lstat) {
+
+  HANDLE handle;
+  FILE_STAT_BASIC_INFORMATION stat_info;
+  FILE_FULL_DIR_INFORMATION* pdir_info;
+  IO_STATUS_BLOCK ioStatusBlock;
+  NTSTATUS status;
+  WCHAR buf[32 * 1024];
+  WCHAR* path_dirpath = NULL;
+  WCHAR* path_filename = NULL;
+  BOOL ret_val = FALSE;
+  DWORD ret_error = 0;
+
+  if (uv__fs_split_path(path, &path_dirpath, &path_filename) != 0) {
+    ret_error = GetLastError();
+    goto cleanup;
```

Comment:
HANDLE looks uninitialized here, but looks like it gets passed to CloseHandle

Reply:
Fixed.